Triggertrap/triggertrap-ios
# Adds the “tickDown” image to the assets catalogue

This pull request fixes #71. Adding the “tickDown” image to the assets catalogue.

Test steps:

- Build and run the app
- Open TimeWarp mode
*Expect** to see a down arrow in the header control
- Tap on the down arrow
**Expect** the panel to animate down, and the arrow animate to point upwards
- Tap on the arrow again
**Expect** the panel to animate up, and the arrow returns to it's original orientation

------

- Turn on Night Mode
- Return to TimeWarp
**Expect** the arrow to be themed correctly
